bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/styles.scss

```diff
@@ -0,0 +1,32 @@
+:root {
+  --color-transparent: #ff000000;
+  ::-webkit-media-controls {
```

Comment:
-webkit-media-controls seems unrecoginzed. Where did you get this from? When I search for it I only see longer forms as in https://gist.github.com/afabbro/3759334

Reply:
It's accurate. https://css-tricks.com/custom-controls-in-html5-video-full-screen/